site/topics/memory/memory.rst

```diff
@@ -15,13 +15,52 @@ Bits and Bytes
 
 
 
-Address Space
-=============
+Random Access Memory (RAN)
+==========================
 
+* Memory provides a way to store data for later use
+* There are several different broad types of memory within a computer, but they all serve the same general purpose
+* In this course, the types of memory that will be covered are
 
+    * Random Access Memory (RAM) --- Stores data and instructions the computer is using
+    * Registers --- Stores data and instructions the CPU is actively using
 
-Addressability
-==============
+
+* For this topic, RAM will be the focus
+
+
+Address Space and Addressability
+--------------------------------
+
+* In a typical computer, RAM stores data the computer is using along with the instructions of the programs being run
+
+    * Technically, instructions are data
```

Comment:
I bet 0 of the students will get just how insanely insane that is. Code _is_ data? That's nutso insane bruh. NUTSO.